fuzzing/testdata/contracts/assertions/assert_allocate_too_much_memory.sol

```diff
@@ -0,0 +1,9 @@
+// This contract attempts to allocate an excessive amount of memory by creating an array with a length of 2^64 causing a panic.
+// PanicCodeAllocateTooMuchMemory = 0x41
+
+contract TestContract {
+    function allocateTooMuchMemory() public {
+        uint256[] memory myArray = new uint256[](2**64); // Allocate too much memory
+        myArray[2**64 - 1] = 42;
+    }
+}
```

Comment:
Small nitpick, but I'd add new lines to the end of all these solidity tests so git doesn't warn about it.